### ISSUE TITLE
Helpers to construct YAML to ease dumping

### ISSUE THREAD

**Issue by fkromer** [crdoconnor/strictyaml#87]
It's possible to [load, modify and dump YAML to a file](https://github.com/crdoconnor/strictyaml/issues/19).

    >>> load(yaml_snippet)
    YAML(OrderedDict([('name', 'Ford Prefect'), ('age', '42'), ('possessions', ['Towel'])]))
    >>> person = load(yaml_snippet, schema)
    >>> person['age'] = 43
    >>> person.as_yaml()

If one wants to construct YAML without loading YAML before using `YAML(...)` to construct YAML is quite inconvenient. Would it be possible to add helpers to help in constructing instances?

**Comment by crdoconnor** [comment]
If I'm reading you correctly, as_document is supposed to do what you want.
Is that right?

**Comment by fkromer** [comment]
Yes. In the meanwhile I've found out how to use `as_document`. [Refer e.g. to usage in the tests](https://github.com/crdoconnor/strictyaml/blob/3148346bc56207c0700cf8d52a1de37840614df6/hitch/story/build-yaml-document-from-scratch.story).

**Comment by crdoconnor** [comment]
Ill make an example in the readme. It's not very obvious where it is right
now.

**Comment by fkromer** [comment]
Right. Thanks!
